components/mpas-ocean/src/shared/mpas_ocn_manufactured_solution.F

```diff
@@ -58,6 +58,7 @@ module ocn_manufactured_solution
    real (kind=RKIND) :: kx, ky
    real (kind=RKIND) :: ang_freq
    real (kind=RKIND) :: eta0
+   real (kind=RKIND) :: viscDel2, viscDel4
```

Comment:
These changes look great. One very minor comment is that these could be set in the init routine.

Reply:
Good point. Done